app/src/main/java/com/android/feedme/ui/component/RecipeFullDisplay.kt

```diff
@@ -201,17 +201,18 @@ fun GeneralInfoDisplay(
               style = TextStyle(fontSize = 16.sp, fontWeight = FontWeight.Medium))
           Text(
               modifier =
-                  Modifier.clickable(
-                      onClick = {
-                        profileViewModel.setViewingProfile(profile)
-                        navigationActions.navigateTo(Screen.PROFILE)
-                      }),
-              text = profile.username,
+                  Modifier.padding(end = 8.dp)
+                      .clickable(
+                          onClick = {
+                            profileViewModel.setViewingProfile(profile)
+                            navigationActions.navigateTo(Screen.PROFILE)
+                          }),
+              text = "@${profile.username}",
               textAlign = TextAlign.Center,
               color = BlueUsername,
               style = TextStyle(fontSize = 16.sp, fontWeight = FontWeight.Medium))
         }
-        Spacer(modifier = Modifier.weight(1f))
+        // Spacer(modifier = Modifier.weight(1f))
```

Comment:
I find the username and the rating really close, why did you remove this spacer ?

<img width="190" alt="image" src="https://github.com/feedme-swent-epfl/feedme-android/assets/100317488/1543540a-39a3-4c3c-9c2a-be867a2db4ab">

Reply:
Yes I think it's better with the Spacer.


> people found a bit weird the way user name and rating was display.

Actually I think people found it weird when there is only the ratins displayed (when no author for the recipe is found), so maybe add the Spacer to the if statement of when the profile username is displayed ?

Reply:
If there is no user name, the rating will be centered like this (i moved the spacer inside the if, like you said).
![image](https://github.com/feedme-swent-epfl/feedme-android/assets/94188707/d4f53370-afc4-4082-b2f5-2377d32a0f7f)

Reply:
I think that's a good solution!